source/adios2/toolkit/transport/file/FileFlexNVMe.h

```diff
@@ -30,9 +32,9 @@ class FileFlexNVMe : public Transport
 {
 
 public:
-    FileFlexNVMe(helper::Comm const &comm);
+    explicit FileFlexNVMe(helper::Comm const &comm);
 
-    ~FileFlexNVMe();
+    ~FileFlexNVMe() noexcept;
```

Comment:
What does explict and noexept do in this context?

Reply:
explicit makes sure that the constructor isnt some type conversion. The linter mostly did this.

noexcept does a compile time check to make sure no exceptions are thrown in the destructor.